src/bindings/app/patch.rs

```diff
@@ -0,0 +1,24 @@
+use rlua::{Lua, UserData, UserDataMethods};
+
+struct PatchParser(patch_rs::PatchParser);
+
+impl UserData for PatchParser {
+    fn add_methods<'lua, M: UserDataMethods<'lua, Self>>(methods: &mut M) {
+        methods.add_method("process", |_, this, _: ()| {
+            Ok(this.0.process().expect("patch_parser error"))
```

Comment:
Shouldnt use `expect`. Should include `rlua::prelude::LuaError`, remove the `Ok` and do `this.0.process().map_err(LuaError::external)` that way if there is an error, it wont panic.